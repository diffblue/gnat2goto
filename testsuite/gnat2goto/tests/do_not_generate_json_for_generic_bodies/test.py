from test_support import *

prove(main='main.adb', specs=['my_pack.ads'])
