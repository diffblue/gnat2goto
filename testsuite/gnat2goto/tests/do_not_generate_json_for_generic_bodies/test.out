[main.assertion.1] line 5 assertion My_Pack.G (I) = 3: SUCCESS
VERIFICATION SUCCESSFUL
